test/integration/infra/scale-testing/deployments/providers.tf

```diff
@@ -0,0 +1,34 @@
+terraform {
+  required_providers {
+    kubernetes = {
+      source  = "hashicorp/kubernetes"
+      version = "2.16.1"
+    }
+    helm = {
+      source  = "hashicorp/helm"
+      version = "2.13.1"
+    }
+  }
+}
+
+data "terraform_remote_state" "eks" {
+  backend = "local"
+
+  config = {
+    path = "../../eks-cluster/state/terraform.tfstate"
+  }
+}
+
+provider "kubernetes" {
+  host                   = data.terraform_remote_state.eks.outputs.cluster_endpoint
+  cluster_ca_certificate = base64decode(data.terraform_remote_state.eks.outputs.cluster_certificate_authority)
```

Comment:
If we intend to scale-test on other clusters beside EKS (GKE and AKS), I think we probably should create variables for cluster-specific values (e.g. `cluster_ca_certificate`) for computing its value based on the cluster type we use. We could refactor in a different PR. We can have an env var to the cluster type for scale-testing in `scale-testing.mk`.